quickstart/src/main/java/com/microsoft/azure/kusto/quickstart/SampleApp.java

```diff
@@ -375,7 +375,7 @@ private static void preIngestionQuerying(ConfigJson config, Client kustoClient)
      * @param tableSchema  Table Schema
      */
     private static void alterMergeExistingTableToProvidedSchema(Client kustoClient, String databaseName, String tableName, String tableSchema) {
-        String command = String.format(".alter-merge table %s %s", tableName, tableSchema);
+        String command = String.format(".alter-merge table %s %s", StringUtils.quoteEntityName(tableName), tableSchema);
```

Comment:
If this method is only used for SampleApp, should it be on the utils files in this package?
(So people downloading it could see how it's implemented)

Reply:
I put it in the main package because we may want to expand its use, and suspect we will (this is an initial step).

They can still see what it does to the table name, but it doesn't have to be duplicated to the quickstart package because the intent of the Sample app is to get them started, not to be too minutiae-oriented.